plugin.audio.itvrs_radio_ice/default.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python
+
+import os
+import xbmcaddon
+import xbmcgui
+
+addon = xbmcaddon.Addon()
+addonname = addon.getAddonInfo('name')
+line1 = 'ITVRS - Radio Ice'
+line2 = '  '
+line3 = 'Use Left Menu To Pause Or Stop'
+
+xbmcgui.Dialog().ok(addonname, line1, line2, line3)
+
+class itvrs_radio_ice:

```

Comment:
A simple function would be better than instantiating a class object just to play a file (plus you need to use xbmcplugin.setresolvedurl as rechi pointed out)